scaleway/helpers_object.go

```diff
@@ -590,3 +592,31 @@ func normalizeOwnerID(id *string) *string {
 
 	return &tab[0]
 }
+
+func addErrorDiagnostic(diags *diag.Diagnostics, err error, resource string, awsResourceNotFoundCode string) (bool, bool) {
```

Comment:
could the function be more object specific in the naming ?
addBucketErrorDiagnostic ?

Reply:
document or name the return values